### ISSUE TITLE
Build fails on node v 0.12.0

### ISSUE THREAD

**Issue by 01bit** [react-bootstrap/react-bootstrap#483]
Cloned the repo and installed all the dependencies using ```npm install``` but on running ```npm run build``` get the following error:
```
/react-bootstrap/tools/build.js:3
import from 'colors';
^^^^^^
SyntaxError: Unexpected reserved word
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Module._extensions..js (module.js:478:10)
    at Object.require.extensions.(anonymous function) [as .js] (/react-bootstrap/node_modules/babel/lib/babel/api/register/node.js:130:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/kushagra/code/ssquealdev/node/PollFilter/node_modules/react-bootstrap/run-babel:5:11)
    at Module._compile (module.js:460:26)
```

Also ```npm test``` fails with this error:
```
ERROR [config]: Invalid config file!
  /react-bootstrap/webpack/test.config.js:1
(function (exports, require, module, __filename, __dirname) { import config fr
                                                              ^^^^^^
SyntaxError: Unexpected reserved word
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Module._extensions..js (module.js:478:10)
    at Object.require.extensions.(anonymous function) [as .js] (/react-bootstrap/node_modules/babel/lib/babel/api/register/node.js:130:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/react-bootstrap/karma.conf.js:4:21)
    at Module._compile (module.js:460:26)
npm ERR! Test failed.  See above for more details.
```

**Comment by mtscout6** [comment]
There is probably something wrong with your system then because those steps work on CI: https://travis-ci.org/react-bootstrap/react-bootstrap/builds/57160553

The CI also does a fresh clone and runs `npm install` then `npm run test` and the last step of `npm run test` is to run `npm run build` https://github.com/react-bootstrap/react-bootstrap/blob/master/package.json#L13.

Are you by chance on Windows 8? I have been hearing about problems with npm install failing to install everything in that system.

**Comment by 01bit** [comment]
Well this is weird. I did a fresh clone of the repo and this time the build is successful but last night it failed on multiple attempts. It might have been an issue with one of the dependencies on npm which has since been fixed or an issue with my system(Mac OSX 10.10.3) or just plain me screwing up. Anyways, apologies for the false alarm.
Edit: More smokescreens :/ The post ```npm test``` build just failed but after another fresh clone the build was again successful. 
P.S Is it possible to get some info on another issue I posted (https://github.com/react-bootstrap/react-bootstrap/issues/466) ? My project is being help up because of this.
cheers
